packages/ui/src/common/components/statistics/TokenValueStat.tsx

```diff
@@ -11,12 +12,17 @@ import { StatisticItem, StatisticItemProps } from './StatisticItem'
 export interface TokenValueStatProps extends StatisticItemProps {
   value?: number | BN | null
   textColor?: string
+  isLoading?: boolean
 }
 
 export const TokenValueStat: FC<TokenValueStatProps> = (props) => {
   return (
     <StatisticItem {...props}>
-      <TotalValue value={props.value} textColor={props.textColor} />
+      {!props.isLoading ? (
+        <TotalValue value={props.value} textColor={props.textColor} />
+      ) : (
+        <Skeleton variant="rect" height="32px" width="50%" />
+      )}
```

Comment:
`TotalValue` extends `TokenValue` and `TokenValue` shows `-` if `value` is `null` or `undefined`.

AFAIK this `-` only shows when the `value` is loading. I think it would be better to render the `<Skeleton />` inside `TokenValue` instead of `-`. In this case the `height` and `width` could be set in the parent components style.
E.g:
```ts
export const TotalValue = styled(TokenValue)<{ textColor?: string }>`
  font-size: 20px;
  line-height: 28px;
  color: ${(props) => props.textColor || Colors.Black[900]};

  ${Skeleton} {
    height: 32px;
    width: 50%;
  }
`
```

WDYT ?